src/Common/TurbulenceClosures/TurbulenceClosures.jl

```diff
@@ -373,7 +427,7 @@ function compute_gradient_flux!(
 end
 
```

Comment:
```suggestion

compute_stress(div_type::WithoutDivergence, ν, S) = (-2 * ν) * S
compute_stress(div_type::WithDivergence, ν, S) = (-2 * ν) * S + (2 * ν / 3) * tr(S) * I

```
This might be a nicer / compact way to do this

Reply:
Also, it allows us to avoid writing the `else` case-- it will just result in a method error.

Reply:
Thanks! I will change it.